src/main/java/steps/ActivitiesSteps.java

```diff
@@ -30,6 +29,12 @@ public void postActivities(Activities activities) {
         response = activitiesClient.postActivities(activities);
     }
 
+    @Step
+    public Response postActivitiesGetResponse(Activities activities) {
+        response = activitiesClient.postActivities(activities);
```

Comment:
There is nothing wrong with this, but you can probably return in-line. 
ie - return activitiesClient.postActivities(activities);